docs/developer-docs/security/rust-canister-development-security-best-practices.md

```diff
@@ -425,6 +425,29 @@ If a canister traps or panics in `pre_upgrade`, this can lead to permanently blo
 
 - See [Current limitations of the Internet Computer](https://wiki.internetcomputer.org/wiki/Current_limitations_of_the_Internet_Computer), section "Bugs in `pre_upgrade` hooks"
 
+### Reinstantiate global timers during upgrades
+
+#### Security Concern
+
+Global timers are deactivated upon changes to the canister's Wasm module. The [IC specification](https://internetcomputer.org/docs/current/references/ic-interface-spec#timer) states this as follows:
```

Comment:
```suggestion
Global timers are deactivated upon changes to the canister's Wasm module. The [IC specification](../references/ic-interface-spec#timer) states this as follows:
```
relative links, please! otherwise they don't get checked by CI